hs_core/hydroshare/utils.py

```diff
@@ -362,9 +362,9 @@ def replicate_resource_bag_to_user_zone(user, res_id):
     if istorage.exists(res_coll):
         bag_modified = istorage.getAVU(res_coll, 'bag_modified')
     if bag_modified == "true":
+        # import here to avoid circular import issue
+        from hs_core.tasks import create_bag_by_irods
         create_bag_by_irods(res_id, istorage)
```

Comment:
@jbradberry This is deliberate to make ```create_bag_by_irods``` call on demand to run synchronously in  ```replicate_resource_bag_to_user_zone()``` as I commented earlier in this PR and will make it run asynchronously later when needed. This PR only addresses the asynchronous download so we can get this deployed first, then I will address bag replication later. Will create an issue so we don't lose it. ```create_bag_by_irods.apply_async()``` is called only in download function, which is in django_irods submodule that you will have to review in conjunction with this PR, take a look at https://github.com/hydroshare/django_irods/pull/12 to review download function in django_irods submodule.